cicd/gitlab/parts/build-datasets-dual-deploy.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# License-related part of the Ensembl/ensembl-genomio CI/CD pipeline
+
+# A generic job to be used by all License jobs with the "extends" mechanism
+# (https://docs.gitlab.com/ee/ci/yaml/#extends)
+
+
+variables:
+  # https://docs.gitlab.com/ee/user/namespace/  
+  CONTAINER_IMAGE_NAME: datasets-cli
+  DATASETS_VERSION: "16.20.0"
```

Comment:
This variable is required to be updated manually when new datasets cli versions are released. At the moment, a separate cron job would check this, and email metazoan email address with a prompt to update it. Outside of this option, its tricky to obtain when checking via curl as gitlab runners don't have curl by default

Reply:
Recent updates to cicd docker yml no longer requires manual intervention to grab and embed the most up to date version. In the end to get this working it required a new stage running ubuntu docker image and passing of variables between jobs using artifacts:reports:dotenv